corehq/apps/userreports/README.rst

```diff
@@ -1422,34 +1424,34 @@ registration:
    {
        "display": "age_range",
        "column_id": "age_range",
-       "type": "conditional_aggregation",
-       "whens": {
-           "0 <= age_at_registration AND age_at_registration < 12": "infant",
-           "12 <= age_at_registration AND age_at_registration < 36": "toddler",
-           "36 <= age_at_registration AND age_at_registration < 60": "preschooler"
+       "type": "integer_buckets",
+       "field": "age_at_registration",
+       "ranges": {
+            "infant": [0, 12],
+            "toddler": [12, 36],
+            "preschooler": [36, 60]
```

Comment:
Because these ranges are inclusive and there's no defined order in which these would be applied, this might be a bad example to include because the ranges overlap

Reply:
good point, cd26e68c15c